CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 
 All notable changes to this project will be documented here.
 
+### v2.1.14
+- fix: allow unseting resources [PR-255](https://github.com/stakater/application/pull/255)
```

Comment:
Better than allowing no resources to be set, this allows arbitrary resources to be configured. This way, if new resource paremeters are added, it'll be possible to set them this way.

The `ephemeralStorage` resource parameter has existed for a while, but it's not possible to set it on the Deployment resource currently. This would fix that. Maybe the change message could be updated to display this broad application to the resources value?

Reply:
Let me know if it fits you!

Reply:
Maybe "arbitrary resource constraints", for added clarity?

Reply:
Done 👍